### PR TITLE
[feature] ActivityCard component

### PR DESCRIPTION
**Pull request by rartner**
* Creating ActivityCard component
* Listing activities on Home using grid-list
* Corrections and new unit tests

## Pics
`Home screen:`
image

## TESTS
* Added unit tests for comparing snapshots
* Correcting unit tests for using `Vuetify`
* Changing scripts for running and updating unit tests (see README)

### CODE REVIEW

File: src/components/ActivityCard.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-container pa-0>
+    <v-card>
+      <v-img
+        :height="imageHeight"
+        :src="imgSrc"
+      ></v-img>
```
Comment (mathvalenza) [inline src/components/ActivityCard.vue]: We could close v-img with "/>". Maybe it fits into one line

File: src/components/ActivityCard.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          </p>
+        </div>
+      </v-card-title>
+      <v-card-actions class="no-padding-top">
+        <v-btn flat>
+          Saber mais
+        </v-btn>
+        <v-spacer></v-spacer>
+        <v-btn :color="buttonColor">
+          {{ finalized ? 'Finalizada' : 'Feito' }}
```
Comment (mathvalenza) [inline src/components/ActivityCard.vue]: We shouldn't use any logic in the template section, so we can use a computed property called "buttonLabel", for example. By the way, what is the difference between "Finalizada" and "Feito" and what is the link with the boolean finalized

File: src/views/Home.vue [hunk 3]
```diff
@@ -5,16 +5,63 @@
...
+          xs12 sm6 md4 lg3 xl3
+          v-for="activity in activities"
+          :key="activity.title"
+        >
+          <activity-card
+            :description="activity.description"
+            :finalized="activity.finalized"
+            :img-src="activity.imgsrc"
+            :title="activity.title"
+          ></activity-card>
```
Comment (mathvalenza) [inline src/views/Home.vue]: />

File: src/components/ActivityCard.vue [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      switch (this.$vuetify.breakpoint.name) {
+        case 'xs': return '150px';
+        case 'sm': return '200px';
+        case 'md': return '200px';
+        case 'lg': return '200px';
+        case 'xl': return '200px';
+        default: console.log('inválid screen size.');
+      }
+
+      return '';
```
Comment (mathvalenza) [inline src/components/ActivityCard.vue]: Using this, maybe we don't need a default case and the console.log could stay here OR we could return ' ' inside the code block in default case

File: src/components/ActivityCard.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+</template>
+
+<script>
+
+export default {
+  name: 'activity-card',
+  props: {
+    description: {
+      type: String,
+      required: true,
```
Comment (mathvalenza) [inline src/components/ActivityCard.vue:41]: In the future, rendering this component will depends on a Promise. We can keep this but the parent component should have and "v-if" condition. Anyway, it is just a comment

**Review by mathvalenza — CHANGES_REQUESTED** [review]

**Comment by rartner** [comment]
[[FEATURE] Card de Atividade (3)](https://trello.com/c/39EDjvED/62-feature-card-de-atividade-3)

**Review by mathvalenza — APPROVED** [review]

**Review by cuzik — APPROVED** [review]
:cow:

**Pull request merged**